website/docs/kubernetes/deploying-a-container-to-kubernetes.md

```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 10
+title: Deploying a container
+description: Deploying a container to Kubernetes 
+keywords: [podman desktop, podman, containers, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Deploying a container to Kubernetes
+
+With Podman Desktop, you can deploy a container to your Kubernetes cluster.
+
+#### Prerequisites
+
+* You registered the Kubernetes cluster in your kubeconfig file: *<your_kubernetes_cluster>*. For example, [Creating a kind cluster](kind/creating-a-kind-cluster.md).
+* Your container, running or stopped, is available on the **Containers** page: *<your_container>*.
+* The Kubernetes namespace to deploy to already exists.
+
+#### Procedure
+
+1. Click **Podman Desktop tray > Kubernetes > Context > *<your_kubernetes_cluster>*** to set your Kubernetes context.
+1. Open **Podman Desktop dashboard > Containers > *<your_container>*** to see the **Container Details** page.
+1. Click <icon icon="fa-solid fa-rocket" size="lg" /> to generate a Kubernetes pod.
```

Comment:
I would mention that this is hidden with containers running on docker. Someone could not see the action and do not understand the reason.

Reply:
Oh: running containers on Podman is a prerequisites then?

Reply:
Atmyes.
There is this rule `hidden="{!(container.engineType === 'podman' &&...}"`